app/screens/UserScreen.tsx

```diff
@@ -150,29 +148,37 @@ export const UserScreen: FC<UserScreenProps> = observer(function UserScreen({
             onPress={() => toggleFollow()}
             style={$profileButton}
           />
+          <View>
+            <Button
+              text={secret ? "Stop private follow" : "Private follow"}
+              onPress={() => togglePrivFollow()}
+              style={$profileButton}
+            />
+            <Text text="Nobody can see your private follow" size="xs" style={$note} />
+          </View>
         </View>
-        <View>
-          <Toggle
-            id="legacy"
-            variant="checkbox"
-            label="Use legacy, unblinded DM's"
-            inputOuterStyle={secret ? $toggleDisabled : $toggle}
-            inputInnerStyle={$toggleInner}
-            inputDetailStyle={$toggleDetail}
-            value={legacy && !secret}
-            disabled={secret}
-            onPress={toggleLegacy}
-          />
-          <Toggle
```

Comment:
why do you get rid of this?   we need to be able to turn it on and off.

Reply:
I've turned toggle to button, it use same function, just difference UI
```js
const togglePrivFollow = async () => {
    await contacts.add({ pubkey: id, legacy, secret: !secret }).catch((e) => console.log(e))
    setSecret(!secret)
}
```

<img width="376" alt="image" src="https://github.com/ArcadeLabsInc/arcade/assets/123083837/0e12708f-b37c-494a-953a-12ef50aac48c">